app/src/main/java/com/example/androidfundamental/UserOverViewActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.androidfundamental
+
+import android.R.attr
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_user_overview.*
+
+
+class UserOverViewActivity : AppCompatActivity() {
+
+    companion object{
+        private val SUB_ACTIVITY_CREATE_USER = 10
+        lateinit var user : User
```

Comment:
This user could be an instance of the class instead of being a companion object.